### DISCUSSION TITLE
Background color after rotation

### DISCUSSION THREAD

**Discussion by Kizuto3** [SixLabors/ImageSharp#2116] [Ideas] [answered]
Is there a way to set background color for an image after rotation? I tried 
`sharpImage.Mutate(x => x.BackgroundColor(ImageSharp.Color.White).Rotate(item.Angle));` 
and 
 `sharpImage.Mutate(x => x.Rotate(item.Angle).BackgroundColor(ImageSharp.Color.White));` 
 but it doesn't seem to work. Seem like image is filled with Color(0, 0, 0) while rotating it. 
![image](https://user-images.githubusercontent.com/38604580/168010209-e7989766-be1f-43e8-9036-7227f722110a.png)

**Comment by JimBobSquarePants** [comment, marked as answer]
You appear to be attempting to manipulate save the image in a format without an alpha channel. You need one for alpha blending to work. 

Also… and I’m getting really tired of repeating myself. This forum channel is for feature suggestions not QA. Please do not misuse this again.

**Reply by fivetaibin** [reply]
I meet the same problem. after rotate the image, Always have a black background color around the image

**Reply by JimBobSquarePants** [reply]
Read the answer above your comment.
